packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx

```diff
@@ -41,9 +41,9 @@ function getValueTagIsPicked(tableValue: ITableSpan[], trace: Trace, nameSelecto
   }
   availableTags = [...new Set(availableTags)];
 
-  const tags = _(availableTags).map('tags').flatten().value();
-  let tagKeys = _(tags).map('key').uniq().value();
-  tagKeys = _.filter(tagKeys, function calc(o) {
+  const tags = _map(availableTags).map('tags').flatten().value();
+  let tagKeys = _map(tags).map('key').uniq().value();
+  tagKeys = _map.filter(tagKeys, function calc(o) {
```

Comment:
We aren't only using the `_map` submodule here. Others are also used like `_uniq`, `_flatter`, `_concat`.

A changed syntax would be something like: 
```tsx
  const tags = _flatten(_map(availableTags, 'tags'));
  let tagKeys = _uniq(_map(tags, 'key'));
  tagKeys = tagKeys.filter(o => o !== nameSelectorTitle);
```